### ISSUE TITLE
Scroll bar hardly visible

### ISSUE THREAD

**Issue by Privatesmee** [hankinsoft/SQLPro#712]
When running multiple queries, the bottom query scroll bar is hardly visible and clickable.

![Screenshot 2021-12-06 at 17 20 15](https://user-images.githubusercontent.com/57482401/144893457-5955522f-0284-4c0c-8b83-5e7fed8b9395.png)

If I run a single query, the scroll bar appears and works correctly.

I am on MACos 12.0.1

**Comment by hankinsoft** [comment]
Could you post your version of SQLPro as well (from the about dialog)? I've worked on this issue in the past and thought I had it sorted out, so I just want to confirm you are on the latest.

**Comment by Privatesmee** [comment]
Apologies for taking so long to respond on this. I am on version 2021.104(build 110401)

**Comment by cwolcott** [comment]
I agree with @Privatesmee .  I see it in SQLPro Studio Version 2021.111 (Build 110875.5) and also SQLPro for Oracle Version 2022.1 (Build 498.5)

**Comment by hankinsoft** [comment]
Hi. I don't currently have a solution for this as I've not been able to reproduce it. I have a few questions/suggestions:

1. What version of macOS is everyone who sees this using?
2. If you toggle your macOS 'show scroll bars' setting to 'Always' do you still see this issue?

**Comment by Privatesmee** [comment]
Hi guys

I'm currently on Monterey 12.5 for Mac and 2022.46 (Build 115559.5) for SQLPro and I am still getting this issue.

Regards

Craig
________________________________

**Comment by cwolcott** [comment]
**OS:** macOS Monterey v12.6.1
**SQL Pro Oracle:** 2022.70 (Build 570.5)

If you look at the video, after running multiple queries the top result area has a horizontal scrollbar, but the bottom result area does not.

After collapsing and restoring the results both have the horizontal scrollbars.

https://user-images.githubusercontent.com/3777526/199144987-f67b7cbb-4633-41de-b675-81b33b039f91.mov

**Comment by Privatesmee** [comment]
Hi guys

Yes collapsing the results area and then showing it does then show the scroll bar for all sections. I shouldn’t have to do that though.

Regards

Craig

Sent from my iPhone

On 1 Nov 2022, at 02:25, cwolcott ***@***.***> wrote:

﻿

OS: macOS Monterey v12.6.1
SQL Pro Oracle: 2022.70 (Build 570.5)

If you look at the video, after running multiple queries the top result area has a horizontal scrollbar, but the bottom result area does not.

After collapsing and restoring the results both have the horizontal scrollbars.

https://user-images.githubusercontent.com/3777526/199144987-f67b7cbb-4633-41de-b675-81b33b039f91.mov<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F3777526%2F199144987-f67b7cbb-4633-41de-b675-81b33b039f91.mov&data=05%7C01%7C%7C63e8f63a60594774679708dabbb05b6b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638028663361247152%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=G5w%2BiEdZF6nd8dEIJJN5OM4e5YPuQRsaou1PZ0KUdUg%3D&reserved=0>

**Comment by cwolcott** [comment]
Totally agree.  I was just trying to give @hankinsoft some insights

**Comment by Privatesmee** [comment]
Apologies

Thank you for looking at this.

Sent from my iPhone

On 1 Nov 2022, at 13:47, cwolcott ***@***.***> wrote:

﻿

Totally agree. I was just trying to give @hankinsoft<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fhankinsoft&data=05%7C01%7C%7C80dc15420ecc4f72780508dabc0fa2cc%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638029072578624066%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=1RkG70rQlOm3sVBt8t4IBwA9N1zP7L8wi4rP9ipj%2BaU%3D&reserved=0> some insights

**Comment by hankinsoft** [comment]
Is this still being seen when using a newer OS such as Venture?

**Comment by cwolcott** [comment]
**macOS:** Tahoe 26.4.1
Still happening in the latest release **Version 2026.97 (Build 111222)**

If you look at the video, after running multiple queries the top result area has a horizontal scrollbar, but the bottom result area does not. After collapsing and restoring the results both have the horizontal scrollbars. See my video [above](https://github.com/hankinsoft/SQLPro/issues/712#issuecomment-1297935134)

**Comment by hankinsoft** [comment]
I've made a few changes and have a build available here:
https://d3fwkemdw8spx3.cloudfront.net/studio/SQLProStudio.2026.98.app.zip

Could you give this a try if you get a chance?

**Comment by cwolcott** [comment]
Nope.  Version 2026.98 (Build 111223) did not fix the issue that I am describing.  To me it is not a big deal just something I also noticed, but this is not my ticket.  Will let @Privatesmee respond.

1. Ran 2 queries so I get 2 result panes
2. I can see the horizontal scrollbar in results pane 1 when moving the mouse left and right
3. I can not see the horizontal scrollbar in results pane 2 when moving the mouse left and right
4. Minimize the result panes
5. Expand the result panes
6. Now I can see the horizontal scrollbar in results pane 1 & 2 when moving the mouse left and right


Here is a video using Version 2026.98 (Build 111223)

https://github.com/user-attachments/assets/aad7c548-2542-43cf-92a4-a572c7eb37e0

**Comment by hankinsoft** [comment]
Qs:

Are you able to reproduce this with queries on any of the sample database?
Are you using a trackball or mouse?
In macOS settings, what do you have set for scroll bar behaviour?

<img width="835" height="737" alt="Image" src="https://github.com/user-attachments/assets/422c2a8a-ae11-46e2-bc6e-a58a74193e16" />

**Comment by cwolcott** [comment]
Again this only happens when there are two or more result panes.  It is always the last pane that is not initially showing the scrollbar when moving the mouse over the data.
Are you able to reproduce this with queries on any of the sample database? Yes (Oracle, SQL Server)
Are you using a trackball or mouse? MOUSE
My current setting is

<img width="480" height="204" alt="Image" src="https://github.com/user-attachments/assets/16a9af6e-4085-4b1d-b466-dfb5eae10f8d" />

https://github.com/user-attachments/assets/0011b2c1-d4cb-4e54-87c7-23cbd2b734f5

If I switch the **Show Scroll Bar** to **Always** you can see the issue.  The scroll bar in the last pane is very small the the padding or margin is incorrect for the last pane.

<img width="460" height="207" alt="Image" src="https://github.com/user-attachments/assets/3ed2a162-a864-4cd4-a152-40660bfbd62d" />

https://github.com/user-attachments/assets/505f819e-437d-40a9-b9da-11238ab7520c

If I close the results pane and re-open then the last panes scrollbar is positioned correctly (better padding or margin)

<img width="569" height="1012" alt="Image" src="https://github.com/user-attachments/assets/452e601b-119d-450d-a1a1-71e350ab9846" />

**Comment by Privatesmee** [comment]
As you can see from my latest screen shot this is still showing a reduced scroll bar in the bottom section. I am using a mouse and these are my settings:

<img width="463" height="201" alt="Image" src="https://github.com/user-attachments/assets/328f9fe9-3c17-4962-93c4-207674fdee1e" />


<img width="1187" height="850" alt="Image" src="https://github.com/user-attachments/assets/9a0350fe-c74f-4bba-840e-bc7258ceb0b8" />

As a side note, the scroll bar becomes fully visible when adjusting the size of the window but when you let go with your cursor, it reverts to being partially hidden.

**Comment by hankinsoft** [comment]
Could you possibly try build https://d3fwkemdw8spx3.cloudfront.net/studio/SQLProStudio.2026.100.app.zip? I believe this sorted the issue for @cwolcott that I had provided for another build and had test a fix for this as well.

**Comment by Privatesmee** [comment]
Yes that has worked for me using the sample DB

**Comment by Privatesmee** [comment]
Do I just use the build you have provided now or will you update what is on the store so that it comes out as an update?

**Comment by Privatesmee** [comment]
I opened another issue for a separate issue so not sure if that has caused this issue or not but wanted to check if this issue didn't introduce this issue or not. Will raise a separate issue if you want me to. 

https://github.com/user-attachments/assets/c289e40a-68db-4bd5-9c7f-b91d1ce96636

Currently using version 2026.102

**Comment by cwolcott** [comment]
Yea I went back to following releases and you could resize the result panes without any issue.

- Version 2026.92 (Build 111217) 
- Version 2026.98 (Build 111223)

The issue does occur in **Version 2026.100 (Build 111225)**.  Not sure about 2026.99 because I don't have that release.

**Comment by Privatesmee** [comment]
Currently using 2026.112 which seems to be showing scrollbars properly. Thank you.
